docs/markdown/recursor/settings.md

```diff
@@ -641,32 +658,37 @@ together with this one to prevent the stack from overflowing.
 ## `max-tcp-clients`
 * Integer
 * Default: 128
+* Available since: 2.9.18
 
 Maximum number of simultaneous incoming TCP connections allowed.
 
 ## `max-tcp-per-client`
 * Integer
 * Default: 0 (unlimited)
+* Available since: 2.9.21
 
 Maximum number of simultaneous incoming TCP connections allowed per client
 (remote IP address).
 
 ## `max-tcp-queries-per-connection`
 * Integer
 * Default: 0 (unlimited)
+* Available since: 4.0.0
```

Comment:
True, dunno what happened, though. I just double-checked my output and seemingly (for whatever reason) that one must've slipped through. I did just amend my PR! :-)